.github/workflows/tests.yaml

```diff
@@ -16,10 +16,15 @@ jobs:
 
     - name: Run shellCheck for tools
       run: |
+          # Run shellCheck 0.8
```

Comment:
the version listed here will become incorrect when a new shellcheck release is published. Maybe something in the lines of:
```
# run shellcheck from the snap which most likely is newer than what the distro provides
```

Reply:
nice, thanks